core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/SQLPlanParser.scala

```diff
@@ -484,7 +488,7 @@ object SQLPlanParser extends Logging {
           case c if c.contains("CreateDataSourceTableAsSelectCommand") =>
             // create data source table doesn't show the format so we can't determine
             // if we support it
-            ExecInfo(node, sqlID, node.name, expr = "", 1, duration = None, node.id,
+            ExecInfo(node, sqlID, normalizedNodeName, expr = "", 1, duration = None, node.id,
```

Comment:
Nit: It is not introduced by this PR, but I do not see we need this case anymore.
It should be handled by the default case, right?

Reply:
You are right. This would be handled by default case. Removed this case statement.